erpnext/accounts/doctype/sales_taxes_and_charges_template/test_records.json

```diff
@@ -2,6 +2,7 @@
  {
   "company": "_Test Company",
   "doctype": "Sales Taxes and Charges Template",
+  "tax_category": "_Test Tax Category 1",
```

Comment:
tax_category is often deliberately left empty. It filters out what can be applied IIRC. 

cc: @deepeshgarg007

Reply:
I went through the code, this fix is invalid. Actually, uniqueness shouldn't be checked for `null` values. 😄